test/lib/ex_aws/s3/download_test.exs

```diff
@@ -0,0 +1,47 @@
+defmodule ExAws.S3.DownloadTest do
```

Comment:
There was an update that needed to be made to `ExAws.S3.Download` for the GenStage upgrade, but I found there was no test coverage. Now there's test coverage 😄.